internal/resources/policy/kind/mutation/recipe/annotation_schema.go

```diff
@@ -0,0 +1,143 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+// nolint: dupl
+package recipe
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/helper"
+	policyrecipecustomcommonmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/custom/common"
+	policyrecipemutationmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/mutation"
+	policyrecipemutationcommonmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/mutation/common"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/kind/common"
+)
+
+var AnnotationSchema = &schema.Schema{
+	Type:        schema.TypeList,
+	Description: "The input schema for custom policy tmc_block_nodeport_service recipe version v1",
+	Optional:    true,
+	ForceNew:    true,
+	MaxItems:    1,
+	Elem: &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			targetKubernetesResourcesKey: common.TargetKubernetesResourcesSchema,
+			scopeKey: {
+				Type:         schema.TypeString,
+				Description:  "Audit (dry-run).",
+				Optional:     true,
+				Default:      "All",
+				ValidateFunc: validation.StringInSlice([]string{"All", "Cluster", "Namespaced"}, false),
+			},
+			AnnotationKey: {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						keyKey: {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						valueKey: {
+							Type:     schema.TypeBool,
```

Comment:
update to `schema.TypeString` as discussed

Reply:
Updated.